### DISCUSSION TITLE
StackStorm to LogicMonitor Integration

### DISCUSSION THREAD

**Discussion by prasadsanjeev** [StackStorm/st2#5916] [General]
Hello, 

Do we have anyone who has successfully integrated stackstorm to LogicMonitor and able to use its functionality ?

I have done integration successfully but still struggling to make it work, unfortunately we do not have enough documentation around it

**Comment by nzlosh** [comment]
You could ask for assistance at the LogicMonitor repository.  https://github.com/StackStorm-Exchange/stackstorm-logicmonitor  It appears there are devs from LogicMonitor who maintain the pack.
